beetsplug/lastgenre/__init__.py

```diff
@@ -575,7 +582,9 @@ def commands(self):
         )
         lastgenre_cmd.parser.set_defaults(album=True)
 
-        def lastgenre_func(lib, opts, args):
+        def lastgenre_func(
+            lib: library.Library, opts: Any, args: list[str]
```

Comment:
`opts` could be typed as `optparse.Values` but my pylint gives me a deprecation warning when I `import optparse`, thus I simply chose this easier way. We have both versions in the codebase BTW.